widgets/AccessInput.php

```diff
@@ -10,32 +10,46 @@
 
 
 use dmstr\db\traits\ActiveRecordAccessTrait;
-use dmstr\modules\redirect\models\ActiveRecord;
 use kartik\select2\Select2;
+use yii\base\Model;
 use yii\base\Widget;
-use yii\widgets\InputWidget;
 use Yii;
+use yii\widgets\ActiveForm;
 
+/**
+ * @package dmstr\widgets
+ *
+ * @property ActiveForm $form
+ * @property Model $model
+ *
+ * @property string $fieldOwner
+ * @property string $fieldDomain
+ * @property string $fieldRead
+ * @property string $fieldUpdate
+ * @property string $fieldDelete
+ * @property string $fieldAppend
+ */
 class AccessInput extends Widget
 {
     public $form;
     public $model;
 
-    public $accessFields = ['owner', 'domain', 'read', 'update', 'delete'];
```

Comment:
This is not in use currently, but actually the idea was that the widget is configurable regarding which fields are shown.